app/src/main/java/com/clearkeep/screen/chat/group_invite/InviteGroupScreen.kt

```diff
@@ -139,4 +249,9 @@ fun SelectedHorizontalBox(selectedItem: List<User>, unSelectItem: (people: User)
             SelectedFriendBox(people = friend, onRemove = { unSelectItem(it) })
         }
     }
-}
\ No newline at end of file
+}
+
+sealed class ManagerMember
```

Comment:
ManagerMember is confuse. MemberUIType is OK?

Reply:
ok fixed